presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -37,6 +37,28 @@ The following file types are supported for the Hive connector:
 * JSON
 * Text
 
+
+Hive Metastore
+--------------
+
+The Hive Metastore is a central metadata repository that Presto uses to access table definitions, partition information, 
+and other structural details about your Hive tables. It serves as the foundation for Presto's Hive connector operations.
+
+Key Concepts
+------------
+- Stores metadata about tables, columns, partitions, and storage locations
+- Enables schema-on-read functionality
+- Supports multiple metastore backends (MySQL, PostgreSQL, AWS Glue)
+
+
+Additional Resources for Metastore Configuration
+------------------------------------------------
+
+* `Metastore Configuration Properties`_ 
+* `How to invalidate metastore cache?`_
```

Comment:
"How to invalidate metastore cache?": This is one of the ways to invalidate the metastore cache and we should update the details in that section. 

The last part of that section
```
Currently, this procedure flushes the cache for all the tables in all the schemas. This is a known limitation and will be enhanced in the future.
```

This enhancement was actually added a few months back. More advanced procedures to invalidate the cache are documented here: https://prestodb.github.io/docs/0.292/connector/hive.html#procedures - `system.invalidate_metastore_cache()` and its variants.